django/db/migrations/operations/utils.py

```diff
@@ -14,6 +14,31 @@ def is_referenced_by_foreign_key(state, model_name_lower, field, field_name):
     return False
 
 
+def describe_operation(operation, backwards):
+    """
+    Returns a string, intended for `stdout.write`, that describes
+    a migration operation.
+    """
+    if hasattr(operation, "code"):
+        action = operation.code.__doc__ if backwards is not True else operation.reverse_code.__doc__
+        prefix = ""
+    elif hasattr(operation, "sql"):
+        action = operation.sql if backwards is not True else operation.reverse_sql
+        prefix = ""
+    else:
+        action = ""
+        prefix = "Undo " if backwards is True else ""
+    if action is None:
+        error = True
+        action = "IRREVERSIBLE"
+    else:
+        action = "{}".format(action.replace("\n", ""))
+        error = False
+    action = action if len(action) < 40 else '{}...'.format(action[:37])
+    styled_action = " -> {}".format(action) if action else action
```

Comment:
Perhaps this would be simpler:
```
if action:
    action = '  -> ` + string
```
I don't think the variable name needs to change.